tests/inputs/date-input-test.jsx

```diff
@@ -17,17 +17,20 @@ describe('DateInput component', () => {
   let handleInputBlur;
 
   const defaults = {
-    inputType: 'date',
+    dateInputName: 'date',
```

Comment:
This file is just unit tests. If you run `npm test` in your terminal window, now they pass. Since you changed the behaviour of `DateInput` component, I adjusted this unit test for the new behaviour.

You don't need to worry about it, I can change them and write new ones myself. As the project grows, I actually thinking of changing my testing strategy and test more business logic and bigger components instead of the very small ones.